### ISSUE TITLE
Proposal: instead of not rendering unknown views, fall back to View and continue to render children 

### ISSUE THREAD

**Issue by bcardarella** [liveview-native/liveview-client-swiftui#1413]
I think our approach to not render unknown views and not rendering children is the wrong approach. I believe instead convert to a `View` or some other generic view, still apply modifiers, and continue rendering children.

This of course is a post v0.3 thing but let's discuss if this is in fact a better approach.

This would be consistent with the web:

<img width="674" alt="Screenshot 2024-08-19 at 7 27 32 PM" src="https://github.com/user-attachments/assets/d7aa8ab1-cf6c-4d93-8ec6-cbde47c83c94">

**Comment by carson-katri** [comment]
I would do this by using `Group` when the element doesn't exist. `Group` automatically applies its modifiers to each child.

We should probably still log the error. It could be something like this:

```html
<nonexistent style="foregroundStyle(.orange);">
  <Text>Hello, world!</Text>
</nonexistent>
```

would render as:

```swift
Group {
  Text("Hello, world!")
}
.foregroundStyle(.orange)
.task {
  logger.log(.warning, "...")
}
```

with a message like this:

```
Unknown element `<nonexistent>`. The app may not render as expected.
```

It would be nice if we had some debug information passed on all elements. I think LiveView does this for a jump-to-definition feature. It could use this info to log the file/line number of the unknown element.

**Comment by bcardarella** [comment]
That works


----------------------
Brian Cardarella
Founder, DockYard

**Comment by carson-katri** [comment]
Currently, this is catching elements such as `<Style>` and `<iframe>`, which seems incorrect. Should we just manually exclude these tag names?

```
'<csrf-token>' is not a known element.
'<Style>' is not a known element.
'<iframe>' is not a known element.
'<nonexistent>' is not a known element.
```

![Simulator Screenshot - iPhone 16 Pro - 2024-11-05 at 11 26 29](https://github.com/user-attachments/assets/9ef6f6b5-2353-4041-8c25-6cc444c4ec32)

**Comment by bcardarella** [comment]
I suspect we'll need to split between HEAD and BODY here to really manage this concern properly. Thoughts?

**Comment by carson-katri** [comment]
Yeah, that makes sense to me. The `iframe` may still need to be handled separately, since I don't think it'd be placed in the head.

```heex
```

**Comment by bcardarella** [comment]
I can change the plug to inject into whatever part of the docuemnt we want, it's really only in `dev` environment so render perf isn't an issue if I were parse the documents and walk into a specific part
